3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -85,6 +85,15 @@ Per i requisiti di sistema e hardware si rimanda al documento #st_v.
 
 
 = Istruzioni d'uso
+== Apertura del programma
+
+Per aprire la pagina web dedicata all'utilizzo di WMS3, è necessario seguire i seguenti passaggi:
+
+  - assicurarsi di avere Docker installato e funzionante sul proprio sistema;
```

Comment:
Toglierei questo punto. Com'è stato detto su Discord, all'utente non interessa di Docker, è una cosa troppo tecnica, e in ogni caso comunque avere Docker è un requisito per l'esecuzione, quindi chi installa il software nei computer dell'ufficio avrà sicuramente installato anche Docker